tests/samples/ReservedWords.scala

```diff
@@ -14,7 +15,7 @@ object ReservedWords {
   type struct_object = native.CStruct2[`match`, native.CInt]
   type struct_finally = native.CStruct2[`def`, `lazy`]
   type union_lazy = native.CArray[Byte, Digit[_1, Digit[_2, _8]]]
-  def `with`(`sealed`: `match`, `implicit`: native.CInt, `forSome`: `lazy`): `type` = native.extern
+  def `with`(`sealed`: `match`, `implicit`: native.Ptr[`match`], `forSome`: `lazy`): `type` = native.extern
```

Comment:
Interesting that this doesn't use the `var` type. Might be another bug?

Reply:
@jonas, I also noticed it, but seems like it's not a bug. Clang says that this type is neither `ConstantArrayType` nor `ArrayType`. Also `getAsString()` method returns `match *`.
But it does not guarantee that bindings will work, it needs to be checked.

The same situation with `implicit` method that has parameter `finally type[12]`. It is converted to a pointer.